content/overview/pricing/managed-services-cost.md

```diff
@@ -9,6 +9,8 @@ aliases:
   - /overview/pricing/other-services/
 ---
 
-Currently, we don’t charge for [managed services]({{< relref "overview/pricing/pricing-terminology.md#managed-services" >}}). We plan to start charging for them in the future to account for the resources they consume and our work to provide them. The pricing for each service will be different, and they will be severable. We will ensure that users of these services are aware of the rates when they are set, and they will not be charged before then.
+Currently, we don’t charge individually for [managed services]({{< relref "docs/apps/managed-services.md" >}}) (services that cloud.gov can spin up quickly and run on your behalf, such as databases, storage, and caching).
 
-This pricing will _only_ be for services created through cloud.gov. Any additional services offered by TTS that are not part of cloud.gov are billed separately, so speak with your 18F Finance point of contact for more information. You can also use external services not managed by cloud.gov or a TTS team at no additional charge.
+We plan to start charging for some of them in the future, to account for the resources they consume and our work to provide them. The pricing for each service will be different, and they will be severable.
```

Comment:
I wonder if we can simplify this a little bit. It is not my intention to start charging for services mid-IAA for a customer. If we do start charging for services, those charges will only kick-in on renewals and new IAAs.

Reply:
Added this info.